pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -191,6 +197,29 @@ func (c *KubeletCollector) parseDeploymentForReplicaset(name string) string {
 	return name[:lastDash]
 }
 
+// parseCronJobForJob gets the cronjob name from a job,
+// or returns an empty string if no parent cronjob is found.
+// https://github.com/kubernetes/kubernetes/blob/b4e3bd381bd4d7c0db1959341b39558b45187345/pkg/controller/cronjob/utils.go#L156
+func (c *KubeletCollector) parseCronJobForJob(name string) string {
```

Comment:
this could be a simple function I guess, it seems we don't need the `KubeletCollector` also would deserve unit tests 🙏

Reply:
good point, on it.